During the building on mock environment, the testing process id 1 is owned
by running user mockbuild's command STUBINIT, though it is owned by root user
on usual environment. The server status does not return ":not_owned".

diff --git a/test/spec_server.rb b/test/spec_server.rb
index 34912b4d..41aadfcf 100644
--- a/test/spec_server.rb
+++ b/test/spec_server.rb
@@ -426,13 +426,13 @@ describe Rack::Server do
     server.send(:pidfile_process_status).must_equal :exited
   end
 
-  it "check pid file presence and not owned process" do
-    owns_pid_1 = (Process.kill(0, 1) rescue nil) == 1
-    skip "cannot test if pid 1 owner matches current process (eg. docker/lxc)" if owns_pid_1
-    pidfile = Tempfile.open('pidfile') { |f| f.write(1); break f }.path
-    server = Rack::Server.new(pid: pidfile)
-    server.send(:pidfile_process_status).must_equal :not_owned
-  end
+  # it "check pid file presence and not owned process" do
+  #   owns_pid_1 = (Process.kill(0, 1) rescue nil) == 1
+  #   skip "cannot test if pid 1 owner matches current process (eg. docker/lxc)" if owns_pid_1
+  #   pidfile = Tempfile.open('pidfile') { |f| f.write(1); break f }.path
+  #   server = Rack::Server.new(pid: pidfile)
+  #   server.send(:pidfile_process_status).must_equal :not_owned
+  # end
 
   it "rewrite pid file when it does not reference a running process" do
     pidfile = Tempfile.open('pidfile') { |f| break f }.path
